File: 8KYU/Quadrants/solution.c
enum quadrants {I=1, II, III, IV};

enum quadrants quadrant(int x, int y) {

    if(x > 0 && y > 0) return 1;
    if(x < 0 && y > 0) return 2;
    if(x < 0 && y < 0) return 3;
    return 4;

}

File: 8KYU/Quadrants/solution.rs
fn quadrant(x: i32, y: i32) -> i32 {
    if(x > 0 && y > 0) {
        return 1;
    }
    if(x < 0 && y > 0) {
        return 2;
    }
    if(x < 0 && y < 0) {
        return 3;
    }
    return 4;
}